### ISSUE TITLE
ozone info

### ISSUE THREAD

**Issue by frolovsa** [jswhit/build_gsinfo#12]
Here are some things i noticed about the ozoneinfo

- [ ] need documentation for what is changing in the ozone info files and what are these choices are based on
- [ ] need to create ozone_info files prior to 1999. Maybe just for noaa11?
- [ ] geos choices are very different from ours. I dont kn

ow where our choices come from? Apparently this also depends on the version of the ozone retrievals.

for reference this is the nasa table for ozone choices for 21c. 
```
SBUV/2   NOAA-14  19970101 19981231              
SBUV/2   NOAA-11  19990101 20000930              
SBUV/2   NOAA-16  20001001 20040930              
SBUV/2   NOAA-17  20020801 20040930              
OMI      Aura     20041001 21001231                
MLS      Aura     20041001 21001231              
OMPS-LP  SNPP     20130101 21001231              
OMPS-NM  SNPP     20130101 21001231              
AIRS     Aqua     20021012 21001231              
IASI     Metop-A  20080801 21001231              
IASI     Metop-B  20130731 21001231              
CrIS-FSR SNPP     20170106 21001231              
CrIS-FSR NOAA-20  20180217 21001231         
```

**Comment by frolovsa** [comment]
This should be revisited after we obtain ozone data from nasa

**Comment by jswhit** [comment]
I don't think we assimilate ozone channels in IASI, CrIS or AIRS.  I will create ozinfo file fragments for the rest based on this list.

**Comment by jswhit** [comment]
For instruments that are still active, my plan is to make their usage consistent with current NCEP ops.  This means we will use significantly less ozone data than NASA 21cr

**Comment by frolovsa** [comment]
Sounds like we should quantify the impact of NCEP vs nasa configuration. 
Should we ask Jack to look into that?
